src/main/kotlin/apply/application/JudgmentService.kt

```diff
@@ -48,17 +63,40 @@ class JudgmentService(
         return LastJudgmentResponse(assignment.pullRequestUrl, judgment.lastRecord)
     }
 
+    fun findExample(userId: Long, missionId: Long): LastJudgmentResponse? {
+        return find(userId, missionId, JudgmentType.EXAMPLE)
+    }
+
+    fun findReal(userId: Long, missionId: Long): LastJudgmentResponse? {
+        return find(userId, missionId, JudgmentType.REAL)
+    }
+
+    private fun find(userId: Long, missionId: Long, judgmentType: JudgmentType): LastJudgmentResponse? {
+        val assignment = assignmentRepository.getByUserIdAndMissionId(userId, missionId)
+        val judgment = judgmentRepository.findByAssignmentIdAndType(assignment.id, judgmentType)
+        return judgment?.let { LastJudgmentResponse(assignment.pullRequestUrl, judgment.lastRecord) }
```

Comment:
```suggestion
        return judgment?.let { LastJudgmentResponse(assignment.pullRequestUrl, it.lastRecord) }
```

개인적으로는 if 문을 활용하는 게 더 보기 좋다고 생각하긴 하는데, 여기 코드 스타일이 있을테니 팀원분들과 잘 논의해서 정하시면 될 것 같습니다!

Reply:
☺️ 논의 후 반영하겠습니다 ~

Reply:
it으로 수정했습니다. 감사합니다!